### PR TITLE
Python 3.4 support

### PR DESCRIPTION
**Pull request by uthunderbird**

### CODE REVIEW

**Comment by kjagiello** [comment]
Awesome! What a coincidence! This is actually something I've started doing yesterday evening (after upgrading the python client to also support python 3.4).

File: setup.py [hunk 1]
```diff
@@ -4,10 +4,16 @@
...
-    'argparse'
+    # 'wsgiref==0.1.2',
+    'argparse',
+    'simplejson' # simplejson is really required for py3 support to avoid encoding problems
 ]
 
+try:
+    import wsgiref
+except ImportError:
+    install_requires.append('wsgiref==0.1.2')
```
Comment (kjagiello) [inline setup.py]: I believe that we can skip the wsgiref requirement completely. I think that tornado required it before and for some reason I have decided to pin the version in our requirements, but it is not needed anymore.

Comment (uthunderbird) [inline setup.py]: It's required for python 2.6 and for python 3.1.x as I understand.

Comment (kjagiello) [inline setup.py]: Yeah, I understand. My point is that the wsgiref package is not needed
anymore for thunderpush to work or am I mistaken?

torsdag 8 oktober 2015 skrev uthunderbird <notifications@github.com>:

Comment (uthunderbird) [inline setup.py]: Think that you're right. You can just attach an additional information for necromancers about that requirement :)

**Pull request merged**